OpenRA.Mods.Common/Traits/World/Pathfinder/CellInfoLayerManager.cs

```diff
@@ -0,0 +1,112 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Drawing;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	public interface ICellInfoLayerManager
```

Comment:
I don't think this interface is buying us anything right now - it's just adding a layer of indirection around the static state we're keeping for reusing all these layers but doesn't actually fix that issue at all.

I think we'd be better off keeping CellInfoLayerManager as a static class for now, until we can do something proper about it.